source/js/components/join/join.jsx

```diff
@@ -327,14 +327,20 @@ export default class JoinUs extends React.Component {
               Email
             </label>
           )}
-          <input
-            name="userEmail"
-            type="email"
-            className="form-control"
-            placeholder={getText(`Please enter your email`)}
-            ref={el => (this.email = el)}
-            onFocus={evt => this.onInputFocus(evt)}
-          />
+          <div
```

Comment:
Why did you need to add a div here?

Reply:
Great question—so this `<input>` needs a label. I could've added `<label>`; however, our `<label>` have `display: inline-block` styling and I didn't want to have to do any style overrides to remove that without having a clear picture of what else that would effect. Simplest way to rememdy this without introducing any style changes was by using a `<div>` instead and adding `aria-label` attr. and `id`.

<img width="1437" alt="Screen Shot 2019-12-05 at 3 43 57 PM" src="https://user-images.githubusercontent.com/36484608/70272506-594cb300-1776-11ea-8e54-c66f5109ab89.png">

Reply:
But why couldn't you just add the "aria-label" directly to the `input` instead of the redirect through `aria-labeledby`?

Reply:
Because I've rarely seen that done 😛 But, I found a use-case for it [here](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html) so, I'll update the PR 👍